### DISCUSSION TITLE
Is it possible to set buf highlight indside builtin select window

### DISCUSSION THREAD

**Discussion by fitrh** [stevearc/dressing.nvim#33] [Q&A] [answered]
![20220330_155709_full](https://user-images.githubusercontent.com/55179750/160782974-e30e1122-99ac-47b1-b87e-6a5a851148d7.jpg)

I want to set highlight for code action source name but I don't know where to hook to modify the window and the item

**Comment by stevearc** [comment, marked as answer]
At the moment there are very limited controls for customizing the UI, because I don't like the idea of supporting an API that diverges from the core `vim.ui.input`. However, if you want to do your own post-processing on the window you should be able to hook into the FileType autocmd.

```lua
function _G.add_highlights()
  ...
end
vim.cmd([[autocmd FileType DressingSelect lua add_highlights()]])
```

**Reply by fitrh** [reply]
I see, thank you, I ended up creating `after/ftplugin/DressingSelect.lua` to do the thing
